Create markdown pages for docs site (#43)
onflow/flow-js-testing: docs/install.md

---
title: Install Flow Javascript Testing Framework
description: How to install the Flow command-line interface (CLI)
---

# Installation

> If you have a non-JS project, you will need to initiate project in your test folder first with:

Reviewer: ```suggestion
If you have a non-JS project, you will need to initiate a project in your test folder first with:
```

```shell
npm init
```

> Then install all necessary packages by running following command in terminal

Reviewer: ```suggestion
Then install all necessary packages by running following command:
```

```shell
npm install flow-js-testing jest @babel/core @babel/preset-env babel-jest @onflow/types
```

> If your project is Javascript based, then run aforementioned command from the folder, where you have `package.json` file.

Reviewer: ```suggestion
If your project *is* JavaScript based, then run the above command from the folder that contains your projects's`package.json` file.
```

### Jest Config

> We need to configure Jest slightly in order for tests to work properly.

Reviewer: ```suggestion
You'll need to configure Jest in order for tests to work properly.
```
